src/touchstone/compare.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import logging
 import json
+import yaml
```

Comment:
i guess we'll need to add pyyaml to [requirements.txt](https://github.com/cloud-bulldozer/touchstone/blob/master/requirements.txt) ?

Reply:
yarp! adding